### ISSUE TITLE
How to set up a windows machine

### ISSUE THREAD

**Issue by adamanteve** [alexa/interactive-adventure-game-tool#15]
I'm not a programmer, just someone who wants to be able to make adventure games. You have instructions to set up a Mac, but not a PC. Do I need a Mac to use this tool?

**Comment by MisterFili** [comment]
yes, i'm told this tool will only run on Mac currently via acloudguru.com

**Comment by brianconway** [comment]
It looks like Package.json was intended to work with Windows based on the last fix note. That would lead me to believe that Windows users should be able to use this as well.

**Comment by brianconway** [comment]
I have managed to get this running in my Surface running Win 10 - It will tell you that it is not compatible when when you complete the NPM Install step, but it launched with NPM Start.

The instructions are the same other than creating the folder and the credentials file. I will update again if I can successfully upload a test

**Comment by ginareams** [comment]
@brianconway Thanks for the info! I'm going to try that out. I was just going in circles for a bit until I started reading the threads in issues. For a moment I was worried it was Mac only, despite the Package.json.

**Comment by Shanker511** [comment]
@brianconway any chance getting a tutorial from you ? on the windows install ? npm keeps crashing for me and npm start will not work. any ideas ?

**Comment by ginareams** [comment]
@Shanker511 You might want to check out this thread to see if anything is helpful.
